ngraph/test/type_prop/dft.cpp

```diff
@@ -0,0 +1,308 @@
+            {2, Dimension(7, 500), 180, 2}, {2}, {2, Dimension(7, 500), 180, 2}, {1, 2}},
+        ConstantAxesAndNoSignalSizeTestParams{{2, Dimension(7, 500), 180, Dimension(1, 18)},
+                                              {2},
+                                              {2, Dimension(7, 500), 180, Dimension(1, 18)},
+                                              {1, 2}},
+        ConstantAxesAndNoSignalSizeTestParams{{2, Dimension(7, 500), Dimension(7, 500), 2},
+                                              {2},
+                                              {2, Dimension(7, 500), Dimension(7, 500), 2},
+                                              {1, 2}},
+        ConstantAxesAndNoSignalSizeTestParams{
+            {2, Dimension(7, 500), Dimension(7, 500), Dimension(1, 18)},
+            {2},
+            {2, Dimension(7, 500), Dimension(7, 500), Dimension(1, 18)},
+            {1, 2}},
+        ConstantAxesAndNoSignalSizeTestParams{
+            {Dimension(0, 2), 180, 180, 2}, {2}, {Dimension(0, 2), 180, 180, 2}, {1, 2}},
+        ConstantAxesAndNoSignalSizeTestParams{{Dimension(0, 2), 180, 180, Dimension(1, 18)},
+                                              {2},
+                                              {Dimension(0, 2), 180, 180, Dimension(1, 18)},
+                                              {1, 2}},
+        ConstantAxesAndNoSignalSizeTestParams{{Dimension(0, 2), 180, Dimension(7, 500), 2},
+                                              {2},
+                                              {Dimension(0, 2), 180, Dimension(7, 500), 2},
+                                              {1, 2}},
+        ConstantAxesAndNoSignalSizeTestParams{
+            {Dimension(0, 2), 180, Dimension(7, 500), Dimension(1, 18)},
+            {2},
+            {Dimension(0, 2), 180, Dimension(7, 500), Dimension(1, 18)},
+            {1, 2}},
+        ConstantAxesAndNoSignalSizeTestParams{{Dimension(0, 2), Dimension(7, 500), 180, 2},
+                                              {2},
+                                              {Dimension(0, 2), Dimension(7, 500), 180, 2},
+                                              {1, 2}},
+        ConstantAxesAndNoSignalSizeTestParams{
+            {Dimension(0, 2), Dimension(7, 500), 180, Dimension(1, 18)},
+            {2},
+            {Dimension(0, 2), Dimension(7, 500), 180, Dimension(1, 18)},
+            {1, 2}},
+        ConstantAxesAndNoSignalSizeTestParams{
+            {Dimension(0, 2), Dimension(7, 500), Dimension(7, 500), 2},
+            {2},
+            {Dimension(0, 2), Dimension(7, 500), Dimension(7, 500), 2},
+            {1, 2}},
+        ConstantAxesAndNoSignalSizeTestParams{
+            {Dimension(0, 2), Dimension(7, 500), Dimension(7, 500), Dimension(1, 18)},
+            {2},
+            {Dimension(0, 2), Dimension(7, 500), Dimension(7, 500), Dimension(1, 18)},
+            {1, 2}}),
+    PrintToDummyParamName());
+
+struct ConstantAxesAndConstantSignalSizeTestParams
```

Comment:
I think we can use one structure instead of `ConstantAxesAndConstantSignalSizeTestParams` and `ConstantAxesAndNoSignalSizeTestParams`

Reply:
Done.